src/precice/impl/DataContext.cpp

```diff
@@ -104,6 +104,27 @@ bool DataContext::hasMapping() const
   return hasReadMapping() || hasWriteMapping();
 }
 
+bool DataContext::isMappingRequired()
+{
+  using namespace mapping;
+  MappingConfiguration::Timing timing;
+
+  if (not hasMapping()) {
+    return false;
+  }
+
+  timing         = mappingContext().timing;
+  bool hasMapped = mappingContext().hasMappedData;
```

Comment:
Little simplification

```suggestion
  if (not hasMapping()) {
    return false;
  }

  auto timing    = mappingContext().timing;
  bool hasMapped = mappingContext().hasMappedData;
```